playbooks/tasks/deploy_vm.yaml

```diff
@@ -24,18 +24,18 @@
 - block:
       - name: "Copy {{ item }} system disk on target"
         copy:
-          src: "{{ vm_file }}"
-          dest: "{{ vm_file_dest }}"
+          src: "{{ hostvars[item].vm_file | default(vm_file) }}"
```

Comment:
This is already done in `deploy_vms_cluster.yaml` playbook

Reply:
okay, but you have to put the behaviour of `vms_disks_directory` in this task now : 
`src: "{{ hostvars[item].vm_disk | default( (vms_disks_directory|default('')) ~ '/' ~ item ~ '.qcow2') }}"`
This is useful to be consistent with the debian branch